uniffi_bindgen/src/interface/attributes.rs

```diff
@@ -272,15 +274,10 @@ impl<T: TryInto<InterfaceAttributes, Error = anyhow::Error>> TryFrom<Option<T>>
     }
 }
 
-// There may be some divergence between Methods and Functions at some point,
-// but not yet.
```

Comment:
Turns out this is the point at which there is divergence between Methods and Functions.